### PR TITLE
Fix the bug #152

### PR DESCRIPTION
**Pull request by zzpbuaa**
When load pdx file it says some structure is missing. Here may be a typo to assign dop to structure.

### CODE REVIEW

File: odxtools/endofpdufield.py [hunk 1]
```diff
@@ -156,8 +156,8 @@ def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase) -> None:
     def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
         """Recursively resolve any short-name references"""
         if self.structure_snref is not None:
-            dops = diag_layer.diag_data_dictionary_spec.data_object_props
-            self._structure = dops[self.structure_snref]
+            structures = diag_layer.diag_data_dictionary_spec.structures
+            self._structure = structures[self.structure_snref]
 
         if self.env_data_desc_snref is not None:
             dops = diag_layer.diag_data_dictionary_spec.data_object_props
```
Comment (andlaus) [inline odxtools/endofpdufield.py:163]: here, the same mistake happens with DOPs instead of the environment data descriptions. can you fix it as well?

**Comment by andlaus** [comment]
Thanks for the contribution! The patch is good, but before I can merge it you need to sign the [CLA](CONTRIBUTORS_LICENSE_AGREEMENT.md). When sending the email, please add me to the CC (andreas.lauser@mercedes-benz.com)

**Comment by andlaus** [comment]
(for MIT licensed projects like odxtools, the purpose of the CLA boils down to you certifying that you created your contributions yourself...)

**Comment by andlaus** [comment]
@zzpbuaa: what's the status here? I'd like to get this bug fixed soonish...

**Comment by zzpbuaa** [comment]
> andreas.lauser@mercedes-benz.com

Sorry for the late reply because I'm so busy these days. I just sent out the CLA. You can check if you received it.

**Comment by andlaus** [comment]
cool! thanks a lot for your contribution. merging.

**Pull request merged**